src/api/OrderAPI.ts

```diff
@@ -25,8 +25,7 @@ import {
   IssuanceOrder,
   SetProtocolUtils,
   SignedIssuanceOrder,
-  TakerWalletOrder,
-  ZeroExSignedFillOrder,
+  FillOrder,
```

Comment:
FillOrder is a confusing name, because it kind of seems like a verb `fillOrder`.

I don't have a better name off the top of my head... but I think the term `ExchangeOrder` is closer to what it is supposed to be... except that TakerWallet isn't really an exchange.

Reply:
probably fine to name it ExchangeOrder for now and just have TakerWallet be a special "Exchange"